### PR TITLE
Remove `icub-common`

### LINKED ISSUE #10: Remove installation of icub-common from the docs
See:
* https://github.com/icub-tech-iit/training-icub-superbuild-sandbox/blob/4b7b0f7db83279abd870c2b7db7c917e4e7417d4/README.md?plain=1#L42
* https://github.com/icub-tech-iit/training-icub-superbuild-sandbox/blob/4b7b0f7db83279abd870c2b7db7c917e4e7417d4/.gitpod.Dockerfile#L29

### PR DESCRIPTION
**Pull request by pattacini**
Fix #10.

### CODE REVIEW

**Comment by traversaro** [comment]
I can't provide this as a suggestion as it is not part of the modified lines, but I guess we need to add:

~~~
sudo bash ./scripts/install_apt_dependencies.sh
~~~

after the clone of the right version of the superbuild to install the dependencies, xref: https://github.com/robotology/robotology-superbuild?tab=readme-ov-file#system-dependencies  .

**Comment by pattacini** [comment]
Sure, I forgot about the replacement!

File: .gitpod.Dockerfile [hunk 1]
```diff
@@ -20,13 +20,6 @@ RUN apt install -y xfce4 xfce4-goodies xserver-xorg-video-dummy xserver-xorg-leg
 # Install python
 RUN apt install -y python3 python3-dev python3-pip python3-setuptools && \
     if [ ! -f "/usr/bin/python" ]; then ln -s /usr/bin/python3 /usr/bin/python; fi
-    
-# Install icub-common
-
-RUN sh -c 'echo "deb http://www.icub.org/ubuntu focal contrib/science" > /etc/apt/sources.list.d/icub.list'
-RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 57A5ACB6110576A6
-RUN apt update
-RUN apt install -y icub-common
```
Comment (pattacini) [inline .gitpod.Dockerfile:29]: We can now make use of `sudo apt install` on Gitpod; hence, installing dependencies will be part of the assignment on Gitpod.

**Comment by pattacini** [comment]
Done, @traversaro

**Review by Nicogene — APPROVED** [review]

**Review by traversaro — APPROVED** [review]

**Pull request merged**